### PR TITLE
Track elapsed ticks and support placing blocks

### PR DESCRIPTION
**Pull request by tyler000000**

### CODE REVIEW

File: backend/src/canvas_state.erl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-module(canvas_state).
+
+-export([start/0]).
+
+
+server_loop(State) ->
+    receive
+        % blocks should be a JSON compatible map
```
Comment (rebecca-millerr) [inline backend/src/canvas_state.erl:8]: Wait isn't it an array of objects? (array of maps I guess) or am I interpreting this wrong

File: backend/sample_websocket_client.html [hunk 2]
```diff
@@ -36,7 +36,6 @@
 		
       function onOpen(evt) {
          writeToScreen("CONNECTED");
-         doSend(JSON.stringify({a: 1}));
```
Comment (rebecca-millerr) [inline backend/sample_websocket_client.html:39]: Ngl I'm not sure how to interface w this from the client :( but code is cute

**Review by rebecca-millerr — APPROVED** [review]
Only comment was on some documentation, would v much love a tutorial on how to cast/call this but seems awesome

**Review by controversial — APPROVED** [review]
banger

**Pull request merged**